index.js

```diff
@@ -5,14 +5,33 @@
 process.chdir(__dirname);
 
 // Perform node version check before loading any other files or modules
-// Doing this check as soon as possible allows us to avoid ES6 parser errors or other issues
+// Doing this check as soon as possible allows us to avoid ES6 parser errors or
+// other issues
+// Try to display messages nicely, but gracefully degrade if anything goes wrong
 var pkg = require("./package.json");
 if (!require("semver").satisfies(process.version, pkg.engines.node)) {
-	/* eslint-disable no-console */
-	console.error("=== WARNING!");
-	console.error("=== The oldest supported Node.js version is", pkg.engines.node);
-	console.error("=== We strongly encourage you to upgrade, see https://nodejs.org/en/download/package-manager/ for more details\n");
-	/* eslint-enable no-console */
+	let colors;
+	let log;
+
+	try {
```

Comment:
I'd really just keep using `console.error` here, I don't see the need to have this elaborate code just to print a message that node needs to be updated.

Reply:
Ooh come on, I had fun doing it, don't take away my fun! :)
I made it extra safe to pass your gates! 🙏